python/cudf/cudf/_libxx/io/json.pyx

```diff
@@ -22,15 +21,8 @@ import collections.abc as abc
 import os
 
 
-def is_file_like(obj):
-    if not (hasattr(obj, 'read') or hasattr(obj, 'write')):
-        return False
-    if not hasattr(obj, "__iter__"):
-        return False
-    return True
-
-
-cpdef read_json(filepath_or_buffer, dtype, lines, compression, byte_range):
+cpdef read_json_libcudf(filepath_or_buffer, dtype,
```

Comment:
can this not just stay as `read_json`?

Reply:
unfortunately not, it collides with the C++ definition in functions.hpp if named `read_json` is there something else we could like to name it to? I'm not crazy about the name either.

Reply:
You should be able to do `from json cimport read_json as cpp_read_json`